src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeCertificateCanonicalNameTests.cs

```diff
@@ -24,12 +24,213 @@ public class Tcp_ClientCredentialTypeCertificateCanonicalNameTests : Conditional
     // Hence, we are only able to determine at runtime whether a particular endpoint presented by the Bridge is going 
     // to pass a variation or fail a variation. 
 
-    [ConditionalTheory(nameof(Root_Certificate_Installed))]
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
+#endif
     [OuterLoop]
-    [MemberData("TcpClientCredentialType_Certificate_With_Only_CanonicalName_MemberData")]
```

Comment:
We had a separate discussion on converting the [Theory] tests to be individual tests in this PR.  Stephen and I disagree about the use of [Theory] tests (there are good arguments on both sides), but I don't disagree strongly enough to alter this PR.  I mention it here only because others may not have heard that conversation.

 I still believe we should have left the original test alone and added new tests #if'ed with FULLXUNIT_NOTSUPPORTED so that moving to Helix would have run the tests as they were originally written.  But okay to continue the PR with these tests refactored like this.